### PR TITLE
Remove old files, restructure files [SOLENG-833]

### PR DESCRIPTION
**Pull request by samuelallan72**
This is to work with the new centrally managed workflow templates.

### CODE REVIEW

File: tests/functional/tests/bundles/overlays/jammy.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+series: jammy
+applications:
+  prometheus:
+    series: focal
```
Comment (gabrielcocenza) [inline tests/functional/tests/bundles/overlays/jammy.yaml.j2:4]: Is it right that prometheus should be on focal in a jammy deployment?

**Review by gabrielcocenza — APPROVED** [review]
I just have one point that would like to double check, but otherwise LGTM

File: tests/functional/tests/bundles/overlays/jammy.yaml.j2 (hunk 1 above)
Comment (samuelallan72) [reply tests/functional/tests/bundles/overlays/jammy.yaml.j2:4]: I don't know, but that's not something I changed, so it's really out of scope for this PR - see [the file on main for reference](https://github.com/canonical/charm-prometheus-blackbox-exporter/blob/7bff4ec5e49bb9dc11c279817e068c0758dd3c53/src/tests/functional/tests/bundles/overlays/jammy.yaml.j2#L1). :)

**Review by jneo8 — APPROVED** [review]
Roughly check the file moving and workflow files.

File: tests/functional/tests/bundles/overlays/jammy.yaml.j2 (hunk 1 above)
Comment (Pjack) [reply tests/functional/tests/bundles/overlays/jammy.yaml.j2:4]: Behalf of @gabrielcocenza to resolve the conversation since it's out of scope in this PR.

**Pull request merged**